src/ui/components/VPNControllerView.qml

```diff
@@ -630,6 +661,33 @@ Item {
             onVisibleChanged: if (visible) col.handleMultilineText()
         }
 
+        RowLayout {
+          id: logoSubtitleOn
+
+          anchors.horizontalCenter: parent.horizontalCenter
+          opacity: 0.8
+          visible: false
```

Comment:
We may not need this line after making the changes to VPNConnectionStability.qml